Add validate OpenTelemetry deployment (#4)
max-cx/openshift-docs: modules/distr-tracing-install-otel-operator.adoc

. Navigate to *Operators* -> *Installed Operators*.

> . On the *Installed Operators* page, select the `openshift-operators` project. Wait until you see that the {OTELName} Operator shows a status of "Succeeded" before continuing.

Reviewer: ```suggestion
. In *Installed Operators*, select the *openshift-operators* project and wait for the {OTELName} Operator status to turn to *Succeeded*.
```
BTW, the Operator is literally listed as **Red Hat OpenShift distributed tracing data collection Operator** in the Operator Hub, correct?

Author: Yes we should change it here

> .Verification

Author: This is similar to https://63349--docspreview.netlify.app/openshift-enterprise/latest/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing

Reviewer: Thank you for the heads up. I will keep this in mind that this procedure is similar to Tempo installation.

Reviewer: Should I just copy the lines from https://62410--docspreview.netlify.app/openshift-enterprise/latest/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing to this file? `[Yes/No]`

Author: No, it was just heads up, nothing needs to be copied

> . Verify that the `status.phase` of  OpenTelemetry collector pod is `Running` and the `conditions` are `type: Ready`.

Reviewer: @pavolloffay, verify where exactly?

Author: The verification should follow after collector CR installation which is missing in this document.

Reviewer: What do you want me to do now? Merge this PR or wait until (I assume) you add the collector CR installation step(s)?

Author: It was not on my list but I can add section for it as well

Reviewer: Is this task part of https://issues.redhat.com/browse/TRACING-3506 or which one?

Reviewer: Or should I just copy the lines from https://63349--docspreview.netlify.app/openshift-enterprise/latest/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing to this file?

Author: It's in the verification / validation so it is https://issues.redhat.com/browse/TRACING-3260

+

> ---
> oc get pod -l app.kubernetes.io/managed-by=opentelemetry-operator,app.kubernetes.io/instance=<namespace>.<instance name> -o yaml
> ---

Reviewer: ```suggestion
[source,terminal]
----
$ oc get pod -l app.kubernetes.io/managed-by=opentelemetry-operator,app.kubernetes.io/instance=<namespace>.<instance name> -o yaml
----
```
